w_common/lib/src/browser/disposable_browser.dart

```diff
@@ -25,8 +25,8 @@ class _InnerDisposable extends disposable_common.Disposable {
   @override
   String get disposableTypeName => '_InnerDisposable';
 
-  Func<Future<Null>> onDisposeHandler;
-  Func<Future<Null>> onWillDisposeHandler;
+  late Func<Future<Null>> onDisposeHandler;
+  late Func<Future<Null>> onWillDisposeHandler;
```

Comment:
Instead of  making these `late` I suggest calling with a `null` aware on line 33 and 38.

Example:
```dart
      return onDisposeHandler?.call();
```

Reply:
Actually, that won't work because it needs to return a future. How about this?
```dart
  @override
  Future<Null> onDispose() async {
    if (onDisposeHandler == null) {
      return;
    }
    return onDisposeHandler!();
  }
```